src/components/ContentWrapper/ContentWrapper.module.scss

```diff
@@ -28,8 +29,15 @@
     }
 
     a {
-      color: var(--color-black);
+      color: var(--color-secondary);
       text-decoration: underline;
+      transition: 0.2s background-color;
```

Comment:
Are we changing `background-color` anywhere that would trigger this transition?

Reply:
Ah! Remnant from a prior test will remove.